moto/ecs/models.py

```diff
@@ -507,14 +516,31 @@ def __init__(
         self.cluster_arn = cluster.arn
         self.name = service_name
         self.status = "ACTIVE"
-        self.running_count = 0
         self.task_definition = task_definition.arn if task_definition else None
         self.desired_count = desired_count
         self.task_sets: List[TaskSet] = []
         self.deployment_controller = deployment_controller or {"type": "ECS"}
         self.events: List[Dict[str, Any]] = []
         self.launch_type = launch_type
         self.service_registries = service_registries or []
+        self.load_balancers = load_balancers if load_balancers is not None else []
+        self.scheduling_strategy = (
+            scheduling_strategy if scheduling_strategy is not None else "REPLICA"
+        )
+        self.platform_version = platform_version
+        self.tags = tags if tags is not None else []
+        self.pending_count = 0
```

Comment:
We can probably improve the logic here as well, as this can now become equal to `desired_count - ecs_running_count` (with a minimum of 0, just in case the user sets the running count larger then the desired count